website/src/components/Chat/ChatListItem.tsx

```diff
@@ -306,7 +306,8 @@ const ChatListItemIconButton = ({ label, onClick, icon }: ChatListItemIconButton
   return (
     <Tooltip label={label}>
       <Box
-        as="button"
+        as="div"
+        role="button"
```

Comment:
Fix this bug 
`validateDOMNesting(...): <button> cannot appear as a descendant of <button>.`